dss/config.py

```diff
@@ -136,11 +136,8 @@ def _get_native_aws_handle() -> typing.Any:
 
     @staticmethod
     def _get_native_gcp_handle() -> typing.Any:
-        if Config.BLOBSTORE_GS_MAX_CUMULATIVE_RETRY is not None:
-            google.resumable_media.common.MAX_CUMULATIVE_RETRY = Config.BLOBSTORE_GS_MAX_CUMULATIVE_RETRY
```

Comment:
Consider setting `MAX_CUMULATIVE_RETRY` to a large value. This may cause the http adapter retry  configuration to take effect first.